binr/rax2/rax2.c

```diff
@@ -16,6 +16,7 @@ static ut64 flags = 0;
 static int use_stdin();
 static int force_mode = 0;
 static int rax(char *str, int len, int last);
+char endline[2]="\n";
```

Comment:
missing spaces around =. and we usually do `const char *endline = "\n";`... and i would probably name this "newline"